owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/drawer/DrawerViewModel.kt

```diff
@@ -47,4 +52,16 @@ class DrawerViewModel(
         useCase = getStoredQuotaUseCase,
         useCaseParams = GetStoredQuotaUseCase.Params(accountName = accountName)
     )
+
+    fun getAccounts(context: Context): List<Account> {
```

Comment:
Use expression body for these 3 methods?

Reply:
Could be an option, for sure.